wiki/Tournaments/GBC/GBC_2022_Spring/en.md

```diff
@@ -217,19 +204,50 @@ Saturday, April 16, 2022
 
 | Team A |  |  | Team B | Match link |
 | --:  |--: | :-- | :-- | :-: |
-| Yzhhのqlll森林汪酱不会大叠 | 5 | **6** | bikabika | [#1](https://osu.ppy.sh/community/matches/99801555) |
-| 啊对对队 | 5 | **6** | 头号倒霉蛋 | [#1](https://osu.ppy.sh/community/matches/99803451) |
-| 金戈铁马的鲜丰水果摊 | **6** | 2 | 可爱的hyahya酱上线啦！！ | [#1](https://osu.ppy.sh/community/matches/99804500) |
-| 啊不对不队 | 3 | **6** | 皮蛋蛋吸手手俱乐部 | [#1](https://osu.ppy.sh/community/matches/99806554) |
+| Yzhhのqlll森林汪酱不会大叠 | 5 | **6** | **bikabika** | [#1](https://osu.ppy.sh/community/matches/99801555) |
+| 啊对对队 | 5 | **6** | **头号倒霉蛋** | [#1](https://osu.ppy.sh/community/matches/99803451) |
+| **金戈铁马的鲜丰水果摊** | **6** | 2 | 可爱的hyahya酱上线啦！！ | [#1](https://osu.ppy.sh/community/matches/99804500) |
+| 啊不对不队 | 3 | **6** | **皮蛋蛋吸手手俱乐部** | [#1](https://osu.ppy.sh/community/matches/99806554) |
 
 Sunday, April 17, 2022
 
 | Team A |  |  | Team B | Match link |
 | --:  |--: | :-- | :-- | :-: |
-| 太美丽了理塘 | **6** | 1 | 青叶摩卡的百合后花园 | [#1](https://osu.ppy.sh/community/matches/99835417) |
-| MyAngel Koishi | **6** | 1 | 你没对象对不队 | [#1](https://osu.ppy.sh/community/matches/99837265) |
-| 货拉拉拉不拉拉布拉多 | **6** | 3 | 涩涩不队 | [#1](https://osu.ppy.sh/community/matches/99840516) |
-| 大怨种被迫当队长 | **6** | 4 | GBC最强阵容9个Miyes | [#1](https://osu.ppy.sh/community/matches/99840241) |
+| **太美丽了理塘** | **6** | 1 | 青叶摩卡的百合后花园 | [#1](https://osu.ppy.sh/community/matches/99835417) |
+| **MyAngel Koishi** | **6** | 1 | 你没对象对不队 | [#1](https://osu.ppy.sh/community/matches/99837265) |
+| **货拉拉拉不拉拉布拉多** | **6** | 3 | 涩涩不队 | [#1](https://osu.ppy.sh/community/matches/99840516) |
+| **大怨种被迫当队长** | **6** | 4 | GBC最强阵容9个Miyes | [#1](https://osu.ppy.sh/community/matches/99840241) |
+
+#### Regular stage week 3
+
+Friday, April 22, 2022
+
+| Team A |  |  | Team B | Match link |
+| --:  |--: | :-- | :-- | :-: |
+| 大怨种被迫当队长 | 1 | **6** | **太美丽了理塘** | [#1](https://osu.ppy.sh/community/matches/99985568) |
+
+Saturday, April 23, 2022
+
+| Team A |  |  | Team B | Match link |
+| --:  |--: | :-- | :-- | :-: |
+| 青叶摩卡的百合后花园 | 3 | **6** | 皮蛋蛋吸手手俱乐部 | [#1](https://osu.ppy.sh/community/matches/100017943) |
+| 涩涩不队 | **6** | 0 | bikabika | [#1](https://osu.ppy.sh/community/matches/100018611) |
+| 可爱的hyahya酱上线啦！！ | **6** | 2 | MyAngel Koishi | [#1](https://osu.ppy.sh/community/matches/100018611) |
+
+Sunday, April 24, 2022
```

Comment:
there should be a colon after all of these

```suggestion
Sunday, April 24, 2022:
```

Reply:
got it , will fix later on later commit